service/src/main/java/com/codeforcommunity/database/tableImpl/CommentTable.java

```diff
@@ -0,0 +1,137 @@
+package com.codeforcommunity.database.tableImpl;
+
+import static com.codeforcommunity.database.tableImpl.PostTable.getRandClapCount;
+import static com.codeforcommunity.database.tableImpl.PostTable.getRandDate;
+
+import com.codeforcommunity.database.table.ICommentTable;
+import com.codeforcommunity.dto.request.CreateCommentRequest;
+import com.codeforcommunity.dto.response.Comment;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * Our implementation of the {@link ICommentTable} in our database. This class will eventually be
+ * replaced once we start actually working with the database, but we'll still use it for testing
+ * purposes.
+ */
+public class CommentTable implements ICommentTable {
+  // A map of an integer (representing a postId) to a Map of that post's comment id to comment.
+  private final Map<Integer, Map<Integer, Comment>> commentMap;
+
+  /** The constructor we'll be using in production. */
+  public CommentTable() {
+    this.commentMap = new HashMap<>();
+  }
+
+  /**
+   * The constructor we'll be using when we want the db to be seeded with fake data.
+   *
+   * @param postCount The count of posts to create
+   */
+  public CommentTable(int postCount) {
+    // Call the other constructor to set up the map.
+    this();
+
+    Random random = new Random();
+
+    for (int i = 0; i < postCount; i++) {
+      // Generate some number of comments for this post in the range of [0, stubPostCount).
+      int commentCount = random.nextInt(PostTable.STUB_POST_COUNT);
+      for (int j = 0; j < commentCount; j++) {
+        // Create comment length like we did for the post body length.
+        int commentLength = random.nextInt(PostTable.LOREM_IPSUM_TEXT.length());
+        // Get comment body like the post body.
+        String commentBody = PostTable.LOREM_IPSUM_TEXT.substring(0, commentLength);
+        // Create the comment.
+        CreateCommentRequest comment =
+            new CreateCommentRequest(String.format("Comment Author %d%d", i, j), commentBody);
+        // Add the comment to our database.
+        this.saveComment(i, comment);
+      }
+    }
+  }
+
+  /**
+   * Get the ID of the most recently inserted item. This is so that we can artificially assign a
+   * valid ID to the next item being inserted. Returns 0 if there are no entries for the given post.
+   *
+   * @return An integer representing the most recent ID.
+   */
+  private int getLastId(int postId) {
```

Comment:
A lot of ORMs and SQL libraries have this as a method so I just threw it in here like this. Do you think it would make sense to have both or just one of them?

Reply:
I ended up converting it to `getNextId()`